tailwind.config.ts

```diff
@@ -262,6 +262,11 @@ export const extendedTheme = {
             "accordion-down": "accordion-down 0.2s ease-out",
             "accordion-up": "accordion-up 0.2s ease-out",
           },
+          hyphens: {
+            auto: "auto",
+            manual: "manual",
+            none: "none",
+          },
```

Comment:
That's only setting the classes up. Cuz TW doesn't have classes for these things. And yes, we're not using it all over, but in this headline I think we really need it to avoid stuff like this:
![image](https://github.com/user-attachments/assets/2db365f8-c25b-4b6f-8ba4-c26811f0dce0)